addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -618,5 +619,16 @@
     </table>
 </template>
 
+<template id="forum_sign_up_cta" name="Sign-Up Call to Action">
+    <div class="o_wforum_sign_up_cta alert alert-info mt-4 collapse show d-none">
+       <h6 class="alert-heading">Enjoying the discussion? Don't just read, join in!</h6>
+       <p>Create an account today to enjoy exclusive features and engage with our awesome community!</p>
+       <div class="d-flex">
+            <a class="forum_register_url btn btn-primary me-2" href="/web/login">Sign up</a>
```

Comment:
Do we have to use the "forum_register_url" mechanism?

Looks like we have a "forum" variable, we could just build the redirect link of login to be its forum.website_url ?